<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>721</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Libros</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagenes/logoLibreria.svg</normaloff>../imagenes/logoLibreria.svg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QTableWidget" name="tblWdgtUsuarios">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>1221</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(0, 0, 0);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Libros</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsertar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>INSERTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>550</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIsbn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Isbn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFechaPub">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Fecha Publicacion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCategoria">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Categoria</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMensaje">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>Generar Repote</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecio">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Precio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTitulo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Titulo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstado">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPortada">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>490</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Portada</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCantidadStock">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Cantidad Stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1421</width>
      <height>871</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/fondo.svg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>tblWdgtUsuarios</zorder>
   <zorder>label</zorder>
   <zorder>btnInsertar</zorder>
   <zorder>btnActualizar</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>txtIsbn</zorder>
   <zorder>txtFechaPub</zorder>
   <zorder>txtCategoria</zorder>
   <zorder>lblMensaje</zorder>
   <zorder>btnReporte</zorder>
   <zorder>txtPrecio</zorder>
   <zorder>txtTitulo</zorder>
   <zorder>txtEstado</zorder>
   <zorder>txtPortada</zorder>
   <zorder>txtCantidadStock</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
